src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,161 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
```

Comment:
remove redundant empty line

Reply:
Hey, thx. This is my usual comment. I'm trying my best to get read of this redundant line